CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to the RGSoC Teams App
+
+* Introduction, thanks for contributing
+* [Code of Conduct](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/CODE_OF_CONDUCT.md)
+
+## Reporting a bug
+
+* Very valuable
+* Check if issue already exists
+* Provide as many details as possible (screenshot)
+
+## Suggesting a new feature
+
+### Labels
+
+* Dibs
+* Ping via comment and/or join Slack
+
+## Resolving an existing issue
+
+### Finding an issue
+
+* See [`beginner-friendly`](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner-friendly) issues for your first steps in open source
+* See [`#pairwithme`](https://github.com/rails-girls-summer-of-code/rgsoc-teams/labels/%23pairwithme) for more involving issues
+* Make sure no one else is already working on it (assignee, comments)
+* Check if issue is still valid (comments)
+
+## Creating a Pull Request
+(inspired by [jbuilder](https://github.com/rails/jbuilder/blob/master/CONTRIBUTING.md))
+
+
+* If related to an issue: better ask first
+* Fork repo
+* Install locally (see [README](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/README.md) for hints)
+* Make sure tests pass locally: `rake spec`
+
+### Create a Topic Branch
+
+Make sure your fork is up-to-date and create a topic branch for your feature or bug fix.
+
+```
+git checkout master
+git pull upstream master
+git checkout -b my-feature-branch
+```
+
+### Write Code
+
+* [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide)
+* With tests (TODO: can be hard for beginners)
```

Comment:
Hm, this is a tricky one. On the one hand I like the idea of beginners being encouraged to try writing tests, and having a rich test suite sets a good example, but can at the same time be super daunting.

Some thoughts on this:

- We could mention our tooling for tests (Rspec, FactoryGirl, Fakerr, etc), maybe link to these?
- We could encourage folks to get in touch for pairing?
- We could link to one of the tests classes that's beginner friendly to give an idea?